diff --git a/cluster/job.batch b/cluster/job.batch
index 1f30995..fbdc94d 100644
--- a/cluster/job.batch
+++ b/cluster/job.batch
@@ -1,11 +1,11 @@
 #!/bin/bash
 #SBATCH --job-name=rl_train
-#SBATCH --output=/home-mscluster/abrickhill/result.txt
+#SBATCH --output=/home-mscluster/abrickhill/RR/cluster/result.txt
 #SBATCH --ntasks=1
 # increase the time here if you need more than 10 minutes to run your job.
 #SBATCH --time=240:00
 #SBATCH --partition=bigbatch
 
-python3 RR/environment/train.py
+python3 ../environment/trainwandb.py
 
-wait;
\ No newline at end of file
+wait;
diff --git a/environment/__pycache__/franka_env.cpython-310.pyc b/environment/__pycache__/franka_env.cpython-310.pyc
index b1aee38..4d980f6 100644
Binary files a/environment/__pycache__/franka_env.cpython-310.pyc and b/environment/__pycache__/franka_env.cpython-310.pyc differ
diff --git a/environment/__pycache__/handover_env.cpython-310.pyc b/environment/__pycache__/handover_env.cpython-310.pyc
index 6acb722..76d9d90 100644
Binary files a/environment/__pycache__/handover_env.cpython-310.pyc and b/environment/__pycache__/handover_env.cpython-310.pyc differ
diff --git a/environment/__pycache__/utils.cpython-310.pyc b/environment/__pycache__/utils.cpython-310.pyc
index bc0be29..8ab4612 100644
Binary files a/environment/__pycache__/utils.cpython-310.pyc and b/environment/__pycache__/utils.cpython-310.pyc differ
